index.html

```diff
@@ -0,0 +1,150 @@
+          <li><a href="#https://www.formula1.com/en/latest.html" class="page-links">News</a></li>
+          <li><a href="https://www.formula1.com/en/latest/tags.race-calendar.2Sa4DEdBSUMcOqa8aYceEe.html" class="page-links campaign" id="campaign">F1 Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+
+    <section id="moto-home">
+      <q class="subtitle">Welcome to the new MotoGP beta website!</q>
+  
+      <h1 class="cocogoose">Monster Energy British grand Prix</h1>
+      <h2 class="year">2023</h2>
+      <p class="annotation">BritishGP <span>TALKING POINTS</span> MotoGP News ...</p>
+      <q class="news">I've always said Aleix has the biggest balls in the paddock</q>
+      <p class="date">2023.08.06(SUN)</p>
+      <p class="address">@MotoGP, British Grand Prix</p>
+    </section>
+
+    <!-- <section id="moto-home">
+      <div class="img-area"></div>
+      <div class="intro">
+        <h3><img src="icons/MotoImg/Flag_of_the_United_Kingdom_(1-2).svg.png" alt=""></h3>
+        <h1>MotoGP Rider Fan Parade</h1>
+        <h2>
+          #BritishGP
+          <span>LIVE:Don't miss any of the Sunday's action at the British GP!</span>
+        </h2>
+        <button>WATCH NOW</button>
+        <h3 class="date">2023.8.6(Sun)</h3>
+        <h4>@MotoGP, British Grand Prix.</h4>
+      </div>
+    </section> -->
+  <!-- main program -->
+
+    <section id="program" class="program">
+      <h2>Main Event</h2>
+      <hr>
+      <article id="exhibition">
+        <img src="icons/MotoImg/superbike.png" alt="Superbike icon">
+        <h3>Exhibition</h3>
+        <p>Watch as our top riders take on the track for a warm up ...</p>
+      </article>
+
+      <article id="workshops">
+        <img src="icons/MotoImg/race-track.png" alt="track icon">
+          <h3>Trackmeet</h3>
+          <p>Stand a chance to win a lap around the track and be part of your team's pit crew ...</p>
+      </article>
+
+      <article id="autograph">
+        <img src="icons/MotoImg/images.png" alt="helmet icon">
+          <h3>Autograph</h3>
+          <p>Get your favorite reacer's signatures. Also meet legends of the track, like Valentino Rossi ...</p>
+      </article>
+
+      <article id="technical">
+        <img src="icons/MotoImg/customer-service.png" alt="support icon">
+          <h3>Technical</h3>
+          <p>A sneak peak into the teams' operations and Meet the team prinicpals of the top teams ...</p>
+      </article>
+
+      <article id="champion">
+        <img src="icons/MotoImg/trophy.png" alt="trophy-icon">
+            <h3>Champions</h3>
+            <p>The F1 trophy will be on display along with some legends and get your own special replica ...</span></p>
+      </article>
+      <a href="#" class="see-races">SEE ALL THE RACES</a>
+      <a href="#" class="tickets">BUY TICKETS NOW!</a>
+    </section>
+  <!-- featured speakers -->
+
+    <section id="drivers">
+      <h2>Our Racers</h2>
+      <hr>
+    </section>
```

Comment:
- [ ] Kindly add some spacing around the following section, to match the given design.

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/JohnteFresh/The-Grand-Prix/assets/62290545/e99bc1e4-481c-4ca2-b4a2-9e2916f8a1a5) | ![image](https://github.com/JohnteFresh/The-Grand-Prix/assets/62290545/ffd2d59b-9c17-453f-8a50-d3cf44a9b64b) |

Reply:
As suggested by the previous reviewer, kindly add some spacing between the speakers as they are too close to each other and not too similar to the deign given 😄 

Your design :
![image](https://github.com/JohnteFresh/The-Grand-Prix/assets/20537712/3d409e2c-1a2d-47fc-865f-14df020fb8dd)

Design given :
![image](https://github.com/JohnteFresh/The-Grand-Prix/assets/20537712/c5b341e7-5ff9-410d-97d8-b60f5b51f7a3)